### PR TITLE
Add recipe for trimal version 1.4.rev15

### PR DESCRIPTION
**Pull request by gregvonkuster**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/trimal/1.4/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package:
+  name: trimal
+  version: "1.4.rev15"
+
+source:
+  git_url: https://github.com/scapella/trimal.git
```
Comment (bgruening) [inline recipes/trimal/1.4/meta.yaml]: @gregvonkuster if possible please use the tarballs and a checksum. This makes it easier to backup the sources on depot.

**Comment by gregvonkuster** [comment]
@bgruening I've changed the recipe to use the more standard url and md5, and I've eliminated the revision information from the version string.  I'm not familiar enough with this package to know if the new revisions are "backward reproducible". Do you think we need to be concerned about this?

**Comment by bgruening** [comment]
I think this is fine. But I don't see your changes.

**Comment by gregvonkuster** [comment]
@bgruening I'm asking the above question to the expert on this package, so I'll post the response as soon as I get it.

**Comment by gregvonkuster** [comment]
Yep - I have merge conflicts because of your change.  I'll try to get things cleaned up.  :(

**Pull request closed without merging**